src/core/views.py

```diff
@@ -1,84 +1,45 @@
 import os
 import json
-import pyotp
+# import pyotp
```

Comment:
There is a comment out left.

Reply:
Thanks.
I removed a comment and push the branch.
Please re-review and approve it.